webapp/src/app/projects/stores/project-invite.state.ts

```diff
@@ -0,0 +1,150 @@
+import { Action, NgxsOnInit, Selector, State, StateContext, Store } from '@ngxs/store';
+import { throwError } from 'rxjs';
+import { catchError, finalize, tap } from 'rxjs/operators';
+import { Logout } from '../../auth/stores/auth.state';
+import { errorToMessage } from '../../shared/util/api-error';
+import { ProjectRole } from '../models/project-role';
+import { ClearCurrentProject, SetCurrentProject } from './projects.state';
+import { ProjectInvite } from '../models/project-invite';
+import { ProjectInviteService } from '../services/project-invite.service';
+
+export class ClearMessages {
+  static readonly type = '[ProjectUser] Clear messages';
```

Comment:
It should be [ProjectInvite] in the message :)

Reply:
done